docs/components/card-catalog.md

```diff
@@ -143,6 +143,35 @@ will default to the following values:
 { pageSize: 15, page: 1 }
 ```
 
+### fetcherCacheKey
+
+The fetcher functionality makes use of SWRV to handle caching of response data. Whenever the cache key is changed the fetcher will automatically
```

Comment:
Add a link to `SWRV` docs: https://docs-swrv.netlify.app/

Reply:
Fixed!